### ISSUE TITLE
cosyvoice3微调求助

### ISSUE THREAD

**Issue by lunar333** [FunAudioLLM/CosyVoice#1718]
我看tools/make_parquet_list.py更新了，instruct有单独文件了，这个文件格式是什么。是key 指令吗？
我原来都是用这种格式：text里面放key 用江西话讲<|endofprompt|>希望省里领导也一样。需要把指令部分放instruct文件吗？<|endofprompt|>去掉？

**Comment by lunar333** [comment]
我后面又看到llm代码的注释，instruct部分还没有实现，那我现在这种格式的微调能生效吗：key 用江西话讲<|endofprompt|>希望省里领导也一样。

**Comment by lunar333** [comment]
<img width="608" height="642" alt="Image" src="https://github.com/user-attachments/assets/c7d40488-e34f-4fc8-ab21-39d6f11b8b42" />

**Comment by lunar333** [comment]
另外，cosycosy3训练时候也有bug，llm.py正确率这 acc = th_accuracy(logits.view(-1, self.speech_token_size + 3), lm_target, ignore_label=IGNORE_ID) 
应该是+200，不是+3 因为我看CosyVoice3LM,定义的是+200，应该是扩充特殊词表了，但是正确率计算没有同步更新过来

**Comment by AntroFyo** [comment]
根据[https://github.com/FunAudioLLM/CosyVoice/pull/1700](url)这个requests，已经修改了llm和flow的模型代码，自测可以跑通。具体repo看这个，[https://github.com/zdj97/cosyvoice-train/blob/main/cosyvoice3/run.sh](url)

**Comment by aluminumbox** [comment]
刚更新了代码，instruct逻辑可以参照新的代码，是不参与流式构造的

**Comment by miningant511** [comment]
我昨天拉了最新的代码，微调cosyvoice3时，llm很好。但是flow微调时，发现dt.forward()参数mu, mask和x维度不一致，多了一个维度，跟指令的引入有关系？

**Comment by wang-b22** [comment]
> 我昨天拉了最新的代码，微调cosyvoice3时，llm很好。但是flow微调时，发现dt.forward()参数mu, mask和x维度不一致，多了一个维度，跟指令的引入有关系？

跟你一样，什么原因啊
